src/Chapter09/Listing09.18.OverridingEqualsAndNotEquals.cs

```diff
@@ -0,0 +1,117 @@
+using System.Runtime.CompilerServices;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_18;
+
+[CompilerGenerated]
```

Comment:
Except for the examples where we are attempting to show what the compiler has generated, we should drop this attribute.
```suggestion
```